### PR TITLE
move css to head, fixes #10

### LINKED ISSUE #10: CSS delay on page load
The initial page render has no style then it flicks to styled when the bundle loads, the <link /> for style should be moved into <head>

### PR DESCRIPTION
**Pull request by earnubs**
Tests to be added once enzyme lands

### CODE REVIEW

File: src/helpers/html.js [hunk 1]
```diff
@@ -16,11 +16,11 @@ export default class Html extends Component {
           {head.title.toComponent()}
           {head.meta.toComponent()}
           {head.link.toComponent()}
+          <link rel="stylesheet" href="/static/style.css" />
```
Comment (bartaz) [inline src/helpers/html.js]: Should/could this be done with helmet? We have {head.link.toComponent()} right above it.

File: src/containers/app.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
 import React, { PropTypes, Component } from 'react';
 import NavLink from '../components/navlink';
+import Helmet from 'react-helmet';
 
 export default class App extends Component {
   render() {
+    // FIXME sstewart 17-Aug-16 move some of this to config
```
Comment (bartaz) [inline src/containers/app.js]: Agreed that some of Helmet props may be better extracted to some config or to Html template, but it can be done later on.

**Comment by bartaz** [comment]
Looks good to me. +1

File: src/containers/app.js (hunk 2 above)
Comment (earnubs) [inline src/containers/app.js]: I quickly looked at nconf, adding this (and card) as a reminder to look more later

**Pull request merged**